app/gui/src/main/scala/org/bitcoins/gui/util/GUIUtil.scala

```diff
@@ -0,0 +1,18 @@
+package org.bitcoins.gui.util
+
+import scalafx.Includes._
+import scalafx.scene.control.TextField
+
+object GUIUtil {
+
+  def setNumericInput(textField: TextField): Unit = {
+    textField.text.addListener {
+      (
+          _: javafx.beans.value.ObservableValue[_ <: String],
+          _: String,
+          newVal: String) =>
+        if (!newVal.matches("\\d*"))
+          textField.setText(newVal.replaceAll("[^\\d]", ""))
+    }
+  }
```

Comment:
So I noticed that this doesn't allow negative signs `-` which someone could in theory want to do during interpolation, just fyi